src/ray/gcs/gcs_server/gcs_health_check_manager.cc

```diff
@@ -14,15 +14,40 @@
 
 #include "ray/gcs/gcs_server/gcs_health_check_manager.h"
 
+#include <string_view>
+
 #include "ray/stats/metric.h"
+
 DEFINE_stats(health_check_rpc_latency_ms,
              "Latency of rpc request for health check.",
              (),
              ({1, 10, 100, 1000, 10000}, ),
              ray::stats::HISTOGRAM);
 
-namespace ray {
-namespace gcs {
+namespace {
+
+std::string_view GetHealthCheckStatus(
```

Comment:
Otherwise we only get integer number, which hard to interpret.
I wish we have [magic enum](https://github.com/Neargye/magic_enum) checked in.

Reply:
auto x =
                  HealthCheckResponse_ServingStatus_Name(HealthCheckResponse::SERVING);